src/Dev/Tasks/FileMigrationHelper.php

```diff
@@ -569,4 +561,30 @@ protected function getFilenameArray()
             ->execute()
             ->map(); // map ID to Filename
     }
+
+    /**
+     * @param File $file
+     * @param array $messages
+     * @param string $legacyFilename
+     *
+     * return void
+     */
+    protected function handleInvalidFile($file, $messages, $legacyFilename)
```

Comment:
does this need to be a protected? could we make it private instead and changes this to a patch release? (optional for patch release, would require a rebase/retarget to `1.6`)

Reply:
Yeah making it private should be no problem. Happy to point it at 1.6, will have a go this afternoon.